src/components/DatePicker.js

```diff
@@ -174,11 +174,17 @@ class DatePicker extends React.Component {
         style={[styles.component, styles.clearFix, this.props.style]}
         tabIndex='0'
       >
-        <div key='componentTop' style={[styles.componentTop, this.state.showCalendar && styles.componentTopOpen]}>
+        <div key='selectedDateWrapper' style={[
+          (this.props.selectedDateWrapperStyle || styles.selectedDateWrapper),
+          this.state.showCalendar && styles.selectedDateWrapperOpen
```

Comment:
I did try that but I wasn't having any luck with the prop style overwriting the component style when I did that.  I would prefer the way you have outlined so perhaps I was just doing something wrong.  I'll play with the order of styles in the array to see if I can get better results.